client/src/js/services/SnisService.js

```diff
@@ -0,0 +1,18 @@
+angular.module('bhima.services')
+  .factory('SnisService', SnisService);
+  
+SnisService.$inject = ['$http', 'util'];
+
+function SnisService($http, util) {  
+  var service = {};
+
+  service.readSnisZs = SnisZs;
+
+  function SnisZs() {
```

Comment:
A better name for this function would be `healthZones()`.  A better URL would be `/snis/health_zones`.  The 'get_' word is implicit in the HTTP verb "GET".